src/routes/+page.svelte

```diff
@@ -39,7 +41,6 @@
 		>
 			<img class="max-h-20" src="{base}/logo.png" alt="splittermond logo" />
 			<span class="loading loading-ring loading-lg my-8 text-primary"></span>
-			<p class="sr-only">{$_('loading_scene')}</p>
```

Comment:
Sadly had to remove the sr-only text here. Since this is only available for a fraction of a second, i think it is fine.

Had to remove it because the i18n-obj is not yet set at this point.